packages/flutter/lib/src/material/tabs.dart

```diff
@@ -25,9 +25,14 @@ import 'tab_indicator.dart';
 import 'text_theme.dart';
 import 'theme.dart';
 
+// M2 Defaults
 const double _kTabHeight = 46.0;
 const double _kTextAndIconTabHeight = 72.0;
 
+// M3 Defaults
+const double _kM3TabHeight = 48.0;
+const double _kM3TextAndIconTabHeight = 64.0;
+
```

Comment:
I did not put these in the tokenized default objects at the bottom of this file because they are used all over this file in different classes, but let me know if y'all have a different preferredApproach. 😜

Reply:
This was on purpose. I was waiting for this preferredSizeFor implementation. I skipped adding tokens that aren't immediately used.